src/main/java/it/pagopa/ecommerce/commons/documents/v2/TransactionAuthorizationRequestedRetriedEvent.java

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.ecommerce.commons.documents.v2;
+
+import it.pagopa.ecommerce.commons.domain.v2.TransactionEventCode;
+import lombok.Generated;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * Business event corresponding to a retry of transaction authorization
+ * requested. This event maps to a transient state for transactions.
+ */
+@Document(collection = "eventstore")
+@Generated
+@NoArgsConstructor
+@ToString(callSuper = true)
+public final class TransactionAuthorizationRequestedRetriedEvent extends TransactionEvent<TransactionRetriedData> {
+
+    /**
+     * Convenience constructor which sets the creation date to now
+     *
+     * @param transactionId          transaction unique id
+     * @param transactionRetriedData retry count data
+     */
+    public TransactionAuthorizationRequestedRetriedEvent(
+            String transactionId,
+            TransactionRetriedData transactionRetriedData
+    ) {
+        super(
+                transactionId,
+                TransactionEventCode.TRANSACTION_AUTHORIZATION_REQUESTED_RETRIED_EVENT,
```

Comment:
I was reading the name of the event. The event is not a retry on the authorization request, but can we see it as a retry on the outcome reception?

Reply:
You're right, but we need to define the point of view of the event to better define its name.
`TRANSACTION_AUTHORIZATION_REQUESTED_RETRIED_EVENT` is compliant with other events defined, but as you said, it is completely wrong in a semantically way. Maybe `TRANSACTION_AUTH_OUTCOME_POLLING_EVENT` is better?